core/src/client/mod.rs

```diff
@@ -12,37 +13,48 @@ pub use self::change_file_content::{
 };
 pub use self::create_file::{create_file, CreateFileError, CreateFileRequest, CreateFileResponse};
 pub use self::delete_file::{delete_file, DeleteFileError, DeleteFileRequest, DeleteFileResponse};
+pub use self::get_file::{get_file, GetFileError, GetFileRequest};
 pub use self::get_updates::{get_updates, FileMetadata, GetUpdatesError, GetUpdatesRequest};
 pub use self::move_file::{move_file, MoveFileError, MoveFileRequest, MoveFileResponse};
 pub use self::new_account::{new_account, NewAccountError, NewAccountRequest, NewAccountResponse};
 pub use self::rename_file::{rename_file, RenameFileError, RenameFileRequest, RenameFileResponse};
+use crate::service::file_encryption_service::EncryptedFile;
+use crate::{API_LOC, BUCKET_LOC};
 
 #[derive(Debug)]
 pub enum ClientError {
```

Comment:
This seems like an anti-pattern, bunching these together. The larger this grows the more things people have to handle, these calls should return their own enums.

Reply:
Unsure what you mean